### ISSUE TITLE
About training by AlexNet

### ISSUE THREAD

**Issue by joei1981** [tflearn/tflearn#477]
When I made a try on "tflearn/examples/images/alexnet.py", I found that for "X, Y = oxflower17.load_data(one_hot=True, resize_pics=(227, 227))", it would load a weight file of "17flowers.pkl" for training. I wonder if I did not use such a file but only use images for training, whether I can obtain a similar trained model compared with the way by "17flowers.pkl"?

**Comment by joei1981** [comment]
Actually, I want to use AlexNet to training  my own data for classification. How should I revise "tflearn/examples/images/alexnet.py" to realize such a purpose ?

**Comment by joei1981** [comment]
Actually, I want to use AlexNet to train my own data for classification. How should I revise "tflearn/examples/images/alexnet.py" to realize such a purpose ?

**Comment by burness** [comment]
you can use the api with image api `tflearn.data_utils.image_preloader`. It has two methods to provide the image data. Here is an example [vgg_network_finetuning.py](https://github.com/tflearn/tflearn/blob/a79d9071ea74b1a995ea6c0d295867f83485e327/examples/images/vgg_network_finetuning.py)

**Comment by joei1981** [comment]
To burness:

Thank you so much for your guidance.
Based on the example of vgg_network_finetuning.py, I have revised alexnet.py and made a successful training model.
However, the training process is very slow, and I am afraid it used cpu. Could you please tell me how to make a configuration if I want to use gpu for training ?

**Comment by burness** [comment]
@joei1981  if you have install tensorflow with gpu, the tflearn will train with gpu.   If you feel to slow you can increase your batch size.

**Comment by joei1981** [comment]
@burness I see. Let me make a try. Thank you.

**Comment by burness** [comment]
@joei1981  are you Chinese?  You can
 add my wechat in http://hacker.duanshishi.com  if you have any problem

**Comment by ibadami** [comment]
Did any one try running /tflearn/examples/images/vgg_network.py ?
When I run the example without any change, I do not get any improvements in accuracy for quite some time.
```
Training samples: 1360
Validation samples: 0
--
Training Step: 500  | total loss: 2.83457
| RMSProp | epoch: 011 | loss: 2.83457 - acc: 0.0542 -- iter: 0864/1360
--
Training Step: 1000  | total loss: 2.83408
| RMSProp | epoch: 023 | loss: 2.83408 - acc: 0.0602 -- iter: 0352/1360
--
Training Step: 1500  | total loss: 2.83363
| RMSProp | epoch: 034 | loss: 2.83363 - acc: 0.0599 -- iter: 1216/1360
--
Training Step: 2000  | total loss: 2.83311
| RMSProp | epoch: 046 | loss: 2.83311 - acc: 0.0628 -- iter: 0704/1360
--
Training Step: 2500  | total loss: 2.83320
| RMSProp | epoch: 058 | loss: 2.83320 - acc: 0.0562 -- iter: 0192/1360
--
Training Step: 3000  | total loss: 2.83377
| RMSProp | epoch: 069 | loss: 2.83377 - acc: 0.0384 -- iter: 1056/1360
--
Training Step: 3500  | total loss: 2.83317
| RMSProp | epoch: 081 | loss: 2.83317 - acc: 0.0704 -- iter: 0544/1360
--
Training Step: 4000  | total loss: 2.83403
| RMSProp | epoch: 093 | loss: 2.83403 - acc: 0.0465 -- iter: 0032/1360
--
Training Step: 4500  | total loss: 2.83315
| RMSProp | epoch: 104 | loss: 2.83315 - acc: 0.0488 -- iter: 0896/1360
--
Training Step: 5000  | total loss: 2.83339
| RMSProp | epoch: 116 | loss: 2.83339 - acc: 0.0571 -- iter: 0384/1360
--
Training Step: 5500  | total loss: 2.83398
| RMSProp | epoch: 127 | loss: 2.83398 - acc: 0.0351 -- iter: 1248/1360
--
```
Is this normal or there is some bug ?

**Comment by muzi-8** [comment]
@burness could you can guide me how to use the ” vgg_network_finetuning.py“，thanks！
